### DISCUSSION TITLE
Training image loading problems

### DISCUSSION THREAD

**Discussion by Chenhanne** [open-mmlab/mmdetection#11195] [General] [answered]
I use a custom coco dataset for training and there are instances where the training images are not fully loaded.
I have 2583 training pictures and 884 test pictures. 
As shown, only half of the training images can be detected. But all the test images can be detected.

![image](https://github.com/open-mmlab/mmdetection/assets/116365899/defba9d7-0b1d-4824-8101-af662701d0e2)

To find the problem, I used the training set as a test set and the test set as a training set to check if it was a problem with the data that caused the images to load incompletely. However, the results are as follows:

![image](https://github.com/open-mmlab/mmdetection/assets/116365899/2c665312-c716-40de-a198-28498367a02e)

![image](https://github.com/open-mmlab/mmdetection/assets/116365899/0eaa88ee-9220-4fcd-9d58-f9c2a2cf8558)

It is easy to see that the original training set was able to be fully loaded (2583 images) after being used as a test set. However, the original test set was only half recognised (442 images) after being used as a training set.
It seems that only the training process has the problem
What's going on here? How can I solve this problem?

**Comment by Chenhanne** [comment]
The model I use is:
_base_ = [
    '../_base_/models/faster-rcnn_r50_fpn.py',
    '../_base_/datasets/coco_detection.py',
    '../_base_/schedules/schedule_1x.py',
    '../_base_/default_runtime.py'
]

**Reply by aso538** [reply, marked as answer]
According to your statement and the corresponding configuration file, if you haven't adjusted the batch size, then it should be that one batch in the training set contains two images
![image](https://github.com/open-mmlab/mmdetection/assets/61680103/217e9b06-b7b8-4c24-94f9-0bead84b98e4)
Just modify the value of batchsize

**Comment by Chenhanne** [comment]
Thank you so much for your answer!!!
















At 2023-12-06 15:41:23, "aso538" ***@***.***> wrote:

According to your statement and the corresponding configuration file, if you haven't adjusted the batch size, then it should be that one batch in the training set contains two images
image.png (view on web)
Just modify the value of batchsize
